app/src/main/res/values/styles.xml

```diff
@@ -8,4 +8,19 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <!-- **************set the rounded drawable as background to your bottom sheet************** -->
```

Comment:
Let's add space between the last * and **set** and same with **sheet**. Something like ** set the... bottom **.
Maybe two spaces. Same thing with the bottom as well.

Reply:
okay i see that now
would do it right now